### PR TITLE
Updated appointment review page.

### PR DESCRIPTION
**Pull request by vbahinwillit**
## Description
Updated appointment review page.

## Testing done
Unit testing

## Screenshots
Screen Shot 2021-03-09 at 12 10 06 PM

## Acceptance criteria
- [ ] Changes are behind va_online_scheduling_cheetah feature flag

- [ ] Appointment type label displays

- [ ] CONTENT - Appointment type label is "COVID-19 vaccine"

- [ ] Date and time of appointment displays

- [ ] Date and time edits are not allowed from this page

- [ ] Facility name displays

- [ ] Clinic name and address displays

- [ ] Facility edits are not allowed from this page

- [ ] Contact information provided by user on previous page displays:

* Email address
* Phone number

- [ ] Contact information edits are allowed from this page

- [ ] Confirm appointment button displays

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/project-cheetah/components/ReviewPage.jsx [hunk 1]
```diff
@@ -42,24 +42,20 @@ function ReviewPage({
     <div>
       <h1>{pageTitle}</h1>
       <p className="vads-u-margin-top--1 vads-u-margin-bottom--4">
-        Please review the information before confirming your appointments.
+        Make sure everything is correct before submitting your request.
```
Comment (jbalboni) [inline src/applications/vaos/project-cheetah/components/ReviewPage.jsx:45]: Is this potentially confusing @laurenernest, saying that you're submitting a request?

Comment (jbalboni) [inline src/applications/vaos/project-cheetah/components/ReviewPage.jsx:45]: Is this potentially confusing @laurenernest, saying that you're submitting a request?

Comment (laurenernest) [reply src/applications/vaos/project-cheetah/components/ReviewPage.jsx:45]: Yes, potentially confusing. The copy in the specs is FPO. Please keep the original description copy for this page. The only content changes/verification is in the AC of the ticket.

Comment (vbahinwillit) [reply src/applications/vaos/project-cheetah/components/ReviewPage.jsx:45]: Changed back to original text.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**